otwcode/otwarchive
# AO3-5518 Create roles for admins

# Pull Request Checklist


## Issue

https://otwarchive.atlassian.net/browse/AO3-5518

## Purpose
Adds roles for admins, and adjusts some functionality of the site to respond to those admin roles.

## Testing Instructions
Check the pages referenced in the tickets to see if the admin's functionality is limited in the ways the ticket suggests. Add/ remove permissions to admin users to test these checks are being enforced.

## References
https://otwarchive.atlassian.net/browse/AO3-5518

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*

(If you have an account on JIRA, please include the same name in the "Full name" field on your JIRA profile,
so we can find you and assign you the issues you're working on.)

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

(If you do not fill this in, we will use your GitHub account name and will use "they" to avoid making assumptions about your gender.)
